ui/raidboss/data/04-sb/dungeon/doma_castle.txt

```diff
@@ -0,0 +1,137 @@
+73.5 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+76.2 "Garlean Fire" sync /:Magitek Rearguard:209E:/
+81.7 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+85.8 "Cermet Pile" sync /:Magitek Rearguard:209D:/
+88.7 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+91.4 "Garlean Fire" sync /:Magitek Rearguard:209E:/
+97.0 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+98.5 "Mines" sync /03:........:Added new combatant Rearguard Mine/
+101.1 "Cermet Pile" sync /:Magitek Rearguard:209D:/
+104.1 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+106.8 "Garlean Fire" sync /:Magitek Rearguard:209E:/
+112.3 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+115.4 "Cermet Pile" sync /:Magitek Rearguard:209D:/
+118.4 "Magitek Ray" sync /:Rearguard Bit:20A1:/
+
+131.1 "Mines" sync /03:........:Added new combatant Rearguard Mine/ jump 68.3
+132.0 "Cermet Pile"
+135.0 "Magitek Ray"
+137.6 "Garlean Fire"
+143.1 "Magitek Ray"
+147.1 "Cermet Pile"
+150.1 "Magitek Ray"
+152.7 "Garlean Fire"
+158.2 "Magitek Ray"
+160.1 "Mines"
+
+### Magitek Hexadrone
+# -ii 20A4 20A6 20A7 2447
+1000 "Start" sync /00:0839:The Training Grounds will be sealed off/ window 1000,0
+1010.6 "Circle Of Death" sync /:Magitek Hexadrone:20A2:/
+1018.9 "2-Tonze Magitek Missile" sync /:Magitek Hexadrone:20A3:/
+1024.1 "Hexadrone Bits" sync /03:........:Added new combatant Hexadrone Bit/
+1029.6 "Bits Activate"
+1039.6 "Circle Of Death" sync /:Magitek Hexadrone:20A2:/
+1045.7 "Hexadrone Bits" sync /03:........:Added new combatant Hexadrone Bit/
+1051.2 "Bits Activate"
+1049.8 "Circle Of Death" sync /:Magitek Hexadrone:20A2:/
+1064.5 "Magitek Missiles" sync /:Magitek Hexadrone:20A5:/
+1068.9 "Hexadrone Bits" sync /03:........:Added new combatant Hexadrone Bit/
+1074.4 "Bits Activate"
+1077.3 "2-Tonze Magitek Missile" sync /:Magitek Hexadrone:20A3:/
+
+1083.9 "Circle Of Death" sync /:Magitek Hexadrone:20A2:/
+1090.0 "Hexadrone Bits" sync /03:........:Added new combatant Hexadrone Bit/
+1095.5 "Bits Activate"
+1098.6 "Magitek Missiles" sync /:Magitek Hexadrone:20A5:/
+1106.4 "2-Tonze Magitek Missile" sync /:Magitek Hexadrone:20A3:/
+1117.9 "Hexadrone Bits" sync /03:........:Added new combatant Hexadrone Bit/
+1123.4 "Bits Activate"
+1119.6 "Magitek Missiles" sync /:Magitek Hexadrone:20A5:/
+1132.3 "Circle Of Death" sync /:Magitek Hexadrone:20A2:/
+1138.7 "Hexadrone Bits" sync /03:........:Added new combatant Hexadrone Bit/
+1144.2 "Bits Activate"
+1146.7 "2-Tonze Magitek Missile" sync /:Magitek Hexadrone:20A3:/
+1156.8 "Magitek Missiles" sync /:Magitek Hexadrone:20A5:/
+1166.2 "Hexadrone Bits" sync /03:........:Added new combatant Hexadrone Bit/
+1171.7 "Bits Activate"
+1174.7 "2-Tonze Magitek Missile" sync /:Magitek Hexadrone:20A3:/
+
+1181.3 "Circle Of Death" sync /:Magitek Hexadrone:20A2:/ jump 1083.9
+1187.4 "Hexadrone Bits"
+1192.9 "Bits Activate"
+1195.0 "Magitek Missiles"
+1203.8 "2-Tonze Magitek Missile"
+1216.0 "Magitek Missiles"
+
+
+### Hypertuned Grynewaht
+# -ii 20A9 20AB 20AC 20AE 20A7 2447
```

Comment:
> 20B0 would be the boss's autoattack, see:
> https://xivapi.com/Action/8368?pretty=true

Is that true? I'm not sure what I see what makes that an auto.  It also only happens much less frequently than I would expect autos to happen.

> make_timeline.py inserted a fair number of --sync-- entries for ID 20B0, during testing I couldn't see anything visible corresponding to this, but it was consistent across pulls. Unsure if these should be removed or not.

Personally, I think these are fine to leave in, as (in this case) they might help the timeline stay "on track" with syncs because they're infrequent and show up before major mechanics.

If you'd like to remove it, can you add it to the `-ii` line too?